src/rules.ruleset

```diff
@@ -61,10 +61,10 @@
       <Rule Id="CA1502" Action="Warning" />          <!-- Avoid excessive complexity -->
       <Rule Id="CA1505" Action="Warning" />          <!-- Avoid unmaintainable code -->
       <Rule Id="CA1506" Action="Warning" />          <!-- Avoid excessive class coupling -->
-      <Rule Id="CA1507" Action="Warning" />          <!-- Use nameof to express symbol names -->
-      <Rule Id="CA1508" Action="Warning" />          <!-- Avoid dead conditional code -->
-      <Rule Id="CA1509" Action="Error" />            <!-- Invalid entry in code metrics rule specification file -->
-      <Rule Id="CA1707" Action="Hidden" />           <!-- Identifiers should not contain underscores -->
+      <Rule Id="CA1507" Action="Error" />            <!-- Use nameof to express symbol names -->
+      <Rule Id="CA1508" Action="Hidden" />           <!-- Avoid dead conditional code --> <!-- TOO MANY FALSE POSITIVES -->
```

Comment:
Do we see ourselves ever enabling this or will it always be hidden? If so, should we just remove the rule?

Reply:
The problem with this  one, is that every 'using' statement became an "error" if any code in the using block did any null check on that value down any code path (which was A LOT). If we remove this rule from this file, then it will get the default value of "Warning". In order to hide it, it must be present.

Reply:
> If we remove this rule from this file, then it will get the default value of "Warning". In order to hide it, it must be present.

Ah, was missing that - assumed excluded=hidden